src/renderers/WebGLMultiRenderTarget.js

```diff
@@ -0,0 +1,65 @@
+import { WebGLRenderTarget } from './WebGLRenderTarget.js';
+
+class WebGLMultiRenderTarget extends WebGLRenderTarget {
+
+	constructor( width, height, count, options ) {
+
+		super( width, height, options );
```

Comment:
The `options` parameter has become pretty messy.
Do you mind removing it from this one?

I prefer this kind of API:

```js
renderTarget = new THREE.WebGLMultiRenderTarget( width, height, 2 );
renderTarget.texture.minFilter = THREE.NearestFilter;
renderTarget.texture.magFilter = THREE.NearestFilter;
renderTarget.texture.type = THREE.FloatType;
```

Reply:
In a previous comment @oskardahlberg said that this approach doesn't work. Is that true?

Reply:
It doesn't work because of the implementation. [The constructor clones the `.texture` to `.textures`](https://github.com/mrdoob/three.js/blob/76dc98ba2de467aadb5afb1b8d92b6b1b7107d08/src/renderers/WebGLMultiRenderTarget.js#L9-L15) and [the cloned textures are used as attachment in `WebGLTextures`](https://github.com/mrdoob/three.js/blob/76dc98ba2de467aadb5afb1b8d92b6b1b7107d08/src/renderers/webgl/WebGLTextures.js#L1181-L1200).

So, `renderTarget.texture.foo = bar` doesn't have an effect. Instead, it should be like

```javascript
renderTarget.textures.forEach(t => {t.foo = bar});
```

If the fact that `renderTarget.texture.foo = bar` has no effect is confusing, we may delete `.texture` after cloning it in the constructor (if possible). `renderTarget.texture` is initialized in the parent `WebGLRenderTarget`.

Reply:
I see. That is indeed confusing...

Can we do this?

```js
class WebGLMultiRenderTarget extends WebGLRenderTarget {

	constructor( width, height, count ) {

		super( width, height );

		this.texture = []; // override texture

		for ( let i = 0; i < count; i ++ ) {

			this.texture[ i ] = new Texture();

		}

	}

	...

}
```

This would be the usage then:

```js
renderTarget = new THREE.WebGLMultiRenderTarget( width, height, 2 );
renderTarget.texture.forEach( function ( texture ) {

	texture.minFilter = THREE.NearestFilter;
	texture.magFilter = THREE.NearestFilter;
	texture.type = THREE.FloatType;

} );
```

Reply:
I guess this approach would map better to how multiple materials are handled in the engine. 

There is no `materials` property but `material` just holds an array of materials.

Reply:
Updated for the rename. The change is bigger than I thought so I would be pleased if you review again.